replica.go

```diff
@@ -43,7 +43,7 @@ type Replica struct {
 	cancel func()
 
 	// Client used to connect to the remote replica.
-	Client ReplicaClient
+	client ReplicaClient
```

Comment:
The `ReplicaClient` no longer has a reference back to its parent `Replica` so it can be set on instantiation and left unexported. Previously, the file replica client required a reference to the DB so it could determine file permissions.